<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="Listener">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>498</y>
      <width>920</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>550</y>
      <width>920</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="DefHint">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>690</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Definition Mode</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionSavePic"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionClear"/>
    <addaction name="actionSelectAll"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionShowHelp"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>外观</string>
    </property>
    <addaction name="actionDarkmode"/>
    <addaction name="actionLightmode"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>加载文件</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>新建文件</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>清空</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>全选</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于...</string>
   </property>
  </action>
  <action name="actionShowHelp">
   <property name="text">
    <string>使用帮助</string>
   </property>
  </action>
  <action name="actionSavePic">
   <property name="text">
    <string>导出图片</string>
   </property>
  </action>
  <action name="actionDarkmode">
   <property name="text">
    <string>深色模式</string>
   </property>
  </action>
  <action name="actionLightmode">
   <property name="text">
    <string>浅色模式</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
